a10_octavia/controller/worker/tasks/vthunder_tasks.py

```diff
@@ -68,28 +68,29 @@ def execute(self, loadbalancer, vthunder):
         try:
             c = self.client_factory(vthunder)
             r = c.slb.virtual_server.delete(loadbalancer_id)
-            status = { 'loadbalancers': [{"id": loadbalancer_id,
-                       "provisioning_status": constants.DELETED }]}
+            status = {'loadbalancers': [{"id": loadbalancer_id,
+                                         "provisioning_status": constants.DELETED}]}
         except Exception as e:
             r = str(e)
             LOG.info(r)
-            status = { 'loadbalancers': [{"id": loadbalancer_id,
-                       "provisioning_status": constants.ERROR }]}
+            status = {'loadbalancers': [{"id": loadbalancer_id,
+                                         "provisioning_status": constants.ERROR}]}
         LOG.info(str(status))
         return status
 
     def revert(self, loadbalancer, *args, **kwargs):
         pass
 
+
 class VThunderComputeConnectivityWait(BaseVThunderTask):
     """"Task to wait for the compute instance to be up."""
 
     def execute(self, vthunder, amphora):
         """Execute get_info routine for a vThunder until it responds."""
         try:
-            
+
             LOG.info("Attempting to connect vThunder device for connection.")
-            attempts = 20
+            attempts = 30
```

Comment:
TODO: future reference - Need to move these numbers to conf file

Reply:
Correct. This is purely a rebase with changes only made in the course of making things work. Not a feature implementation.